gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -56,7 +82,7 @@ def switch(self, an_input: str):
 
     def call_help(self, _):
         """Process Help command"""
-        self.print_help(self.current_coin)
+        self.print_help()
```

Comment:
I think as an argument this should take `self`

Reply:
Since I dropped `@staticmethod`, `print_help` is now just a class method.

Reply:
Yea, I think you're right!